modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Improved
+- The `editor.selection.scrollIntoView()` API now accepts no element to instead scroll the current selection into view #TINY-7291
```

Comment:
```suggestion
- The `editor.selection.scrollIntoView()` API now scrolls the current selection into view if no element is provided #TINY-7291
```
Not sure exactly how to word it, but we should try to make it clear that this is an alternative to the current behaviour, not a replacement.

Reply:
This is the opposite of what we want to be saying, so I don't like this suggestion sorry.

Reply:
```suggestion
- The `element` argument of the `editor.selection.scrollIntoView()` API is now optional, and if it is not provided the current selection will be scrolled into view instead #TINY-7291
```

Reply:
That seems fine if we remove the `instead` at the end?

Reply:
updated